app/models/manageiq/providers/lenovo/physical_infra_manager/parsers/parser.rb

```diff
@@ -0,0 +1,268 @@
+# Class that provides methods to parse LXCA data to ManageIQ data
+# This class provides only data present in all supported versions
+# Data specific of some version must be parsed on subclasses
+module ManageIQ::Providers::Lenovo
+  class Parser
+    require_relative 'parser_dictionary_constants'
```

Comment:
@douglasgabriel Once it has different client version, doesn't make this line being required inside the versioned parser? If some information meaning (e.g.: `POWER_STATE_MAP` codes) change in the LXCA REST API, in a future version, it can break the parser.

Reply:
Agree. So I've created a method on specific parser that provides the specific dictionary. See [parser_1_3.rb](https://github.com/ManageIQ/manageiq-providers-lenovo/pull/98/files#diff-3c836210612c4d26299119496e46dd2d)

Reply:
Sounds good! :D